v2/emailpassword/serverless/with-aws-lambda/about.mdx

```diff
@@ -4,22 +4,23 @@ title: About
 hide_title: true
 ---
 
-<!-- COPY DOCS -->
+<!-- COPY SECTION -->
 <!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/about.mdx -->
+<!-- 1 -->
 
 # About
 
 ## Overview steps
 
 1. Frontend setup
 2. Backend setup
-3. Creating an auth serverless function to expose SuperTokens APIs to the frontend
-4. AWS API Gateway setup
-5. Using SuperTokens functions in your APIs (for example for session verification).
-6. Next steps
+3. Using SuperTokens functions in your APIs (for example for session verification).
+4. Next steps
 
 ## [An example GitHub repo](https://github.com/supertokens/supertokens-auth-react/tree/master/examples/with-aws-lambda) 
 
+<!-- END COPY SECTION -->
+
 :::important
 This example follows the "emailpassword" recipe.
 :::
```

Comment:
the "this" example refers to the example app, and that is in thirdpartyemailpassword. So maybe change this text to say "The example app follows the thirdpartyemailpassword recipe, however this section in the docs follows the emailpassword recipe, so continue reading on."

And make similar changes to other recipes.